main.go

```diff
@@ -97,6 +97,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	feedbackControllerURL, err := url.Parse(cfg.FeedbackControllerURL)
```

Comment:
If i understand it correctly, the feedbackControllerURL is only required if FeedbackEnabled is true. Maybe we can wrap this URL parsing logic with:
```
if cfg. FeedbackEnabled {
 ...
}
```